robot/demos/SkidSteerServos/SkidSteerServos.ino

```diff
@@ -0,0 +1,107 @@
+#include <ArduinoBlue.h>
+
+#include <SoftwareSerial.h>
+
+#include <Servo.h>
+
+Servo myservo;
+Servo myservo1;
+Servo myservo2;
+Servo myservo3;
+Servo myservo4;
+Servo myservo5;
+
+
+int prevThrottle = 49;
+int prevSteering = 49;
+int throttle, steering;
+// Initialize Left & Right Velocity variables to Rest
+int vr = 90;
+int vl = 90;
+SoftwareSerial bluetooth(7, 8);
+
+ArduinoBlue phone(bluetooth);
+
+void setup() {
+  // Attach Servos to pins:
+  // Right side servos
+  myservo.attach(9);  
+  myservo1.attach(10);  
+  myservo2.attach(11);
+  
+  //Left side Servos
+  myservo3.attach(3);
+  myservo4.attach(5);
+  myservo5.attach(6);
+
+
+  // initialize serial communications at 9600 bps:
+  Serial.begin(9600); 
+  bluetooth.begin(9600);
+
+  //activate pull-up resistor on the push-button pin
+  pinMode(buttonPin, INPUT_PULLUP); 
+  
+  Serial.println("setup complete");
+}
+
+float right = 90;
+float left = 90;
+float deg = 0;
+
+
+// New map function that supports float variables.
+ float mapFloat(float x, float in_min, float in_max, float out_min, float out_max) {
+  return (x - in_min) * (out_max - out_min) / (in_max - in_min) + out_min;
+}
+void loop() {
+  
+ // Lead Velocity Value from bluetooth controller. Values range from 0 to 90
+  throttle = phone.getThrottle();
+ // Steering Value from bluetooth controller. Values range from 0 to 90
+  steering = phone.getSteering();
+ 
+ // New Rest Value is 0
+  throttle -= 49;
+  steering -= 49;
+     
+  delay(200); // add some delay between reads
```

Comment:
Just a heads up that when you move over to the real project, we'll be using encoders which will be triggered by interrupt service routines, so we won't be able to use delay() anymore. The classic way of getting around this is by using millis() or micros()... Teensy makes things even easier with the elapsedMillis (possibly elapsedMicros too?) construct.

Reply:
How about `_delay_ms()` from including `<util/delay.h>`? :eyes:

Reply:
doesn't that do the same thing? I think the only way around it is to do the classic:
```
  unsigned int prevRead = millis();
  if(millis()-prevRead > 200){
    // do thing
  }
```
which can be written as the following with the proper library (or on a Teensy it comes built in):
```
  elapsedMillis prevRead;
  if(prevRead > 200){
    // do thing
    prevRead = 0;
  }
```